kotlintest-extensions/kotlintest-extensions-spring/src/main/kotlin/io/kotlintest/spring/SpringListener.kt

```diff
@@ -1,22 +1,50 @@
 package io.kotlintest.spring
 
 import io.kotlintest.Spec
+import io.kotlintest.TestCase
+import io.kotlintest.TestResult
 import io.kotlintest.extensions.ConstructorExtension
 import io.kotlintest.extensions.TestListener
+import io.kotlintest.extensions.TopLevelTest
 import org.springframework.beans.factory.config.AutowireCapableBeanFactory.AUTOWIRE_CONSTRUCTOR
 import org.springframework.test.context.TestContextManager
 import kotlin.reflect.KClass
 import kotlin.reflect.full.primaryConstructor
 
 object SpringListener : TestListener {
 
+  // Necessary to keep specs and contexts in a map because this Listener
```

Comment:
The comment doesn't make sense

Reply:
I think I erased the next line. oops.
This map is needed because each spec needs its own context. But as this listener is a singleton, we can't associate each listener to each spec.

To avoid breaking code, I added this map to keep those separate contexts